src/plugins/intel_gpu/src/graph/include/primitive_type_base.h

```diff
@@ -55,7 +55,7 @@ struct primitive_type_base : primitive_type {
                                         "[GPU] Original name: ", p->origin_op_name, "\n"
                                         "[GPU] Original type: ", p->origin_op_type_name, "\n"
                                         "[GPU] Reason: ", e.what());
-            throw ov::Exception(ss.str());
+            OPENVINO_UNREACHABLE(ss.str());
```

Comment:
Can we have a macro with better name for such cases? "Unreachable" sounds misleading.
OPENVINO_THROW maybe?

Reply:
Will do it in the separate PR :)